sitespeed.io/scripts/machines.js

```diff
@@ -0,0 +1,10 @@
+const { constructURL } = require("../utils");
+
+const TIMEOUT = 120000;
```

Comment:
This is only used in 1 place. Was your intention to make this clearer and improve the readability?